tools/lou_maketable.d/submit_rows.py

```diff
@@ -34,18 +34,18 @@ def __next__(self):
         if not row or row[0].startswith("#"):
             return next(self)
         if not len(row) == 3:
-            printerrln('expected 3 columns, got %s: %s' % (len(row),row))
+            printerrln('expected 3 columns, got {}: {}'.format(len(row),row))
```

Comment:
```suggestion
```suggestion
            printerrln(f'expected 3 columns, got {len(row)}: {row}')
```
```